core/io/resource.cpp

```diff
@@ -200,6 +200,57 @@ void Resource::reload_from_file() {
 	copy_from(s);
 }
 
+void _dupe_sub_resources(Variant &v, Node *p_for_scene, HashMap<Ref<Resource>, Ref<Resource>> &remap_cache) {
```

Comment:
```suggestion
void _dupe_sub_resources(Variant &r_variant, Node *p_for_scene, HashMap<Ref<Resource>, Ref<Resource>> &p_remap_cache) {
```
Also you should make it private method of Resource. We don't use global functions often, especially when there is no need.

Reply:
So this is what https://github.com/godotengine/godot/pull/71578#discussion_r1333500876 was for? 🙃

It's better to move it to a method.

Reply:
Yep, it makes sure the name isn't accessible outside this file, causes clashes otherwise

Reply:
Made them private methods. Also I believe it should be `r_remap_cache` (and further down `r_resources_found`) since you must pass the HashMap and it gets modified by the method?